### PR TITLE
#2 preparation before 14.03.2023

### PR DESCRIPTION
**Pull request by InnaLazarenko**

### CODE REVIEW

File: .github/workflows/nextjs.yml
Comment (var-bin) [inline .github/workflows/nextjs.yml:1]: This file had been deleted in one of the previous merges. It shouldn't be presented in our project anymore. Please, remove it

File: constants/inter-latin.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { Inter } from '@next/font/google';
+
+export const inter = Inter({
+  subsets: ['latin'],
```
Comment (var-bin) [inline constants/inter-latin.ts:4]: As far as I see ([styleguide](https://www.figma.com/file/Pp48CPTdPVHY3Fkrxqn45b/Varbin-Tech_?node-id=2-127&t=yEX9oQA52QxcVppU-0)), we need to add `weight`:
- 400
- 500
- 700

File: darkTheme.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { createTheme } from '@mui/material/styles';
+import { themeMuiButtonRoot, themeTypography } from 'lightTheme';
```
Comment (var-bin) [inline darkTheme.ts:2]: Please, add `./`in front of `lightTheme`. Please, separate different imports with empty line

File: lightTheme.ts [hunk 3]
```diff
@@ -0,0 +1,163 @@
+import { createTheme } from '@mui/material/styles';
+
+import { inter } from './constants/inter-latin';
+
+export const themeTypography = {
+  fontFamily: inter.style.fontFamily,
+  h1: {
+    fontSize: '3.5em',
+    fontWeight: '700',
```
Comment (var-bin) [inline lightTheme.ts:9]: Values without units shouldn't be quoted:
`fontWeight: '700',` -> `fontWeight: 700,`

File: pages/_app.tsx [hunk 4]
```diff
@@ -19,6 +23,12 @@ export interface MyAppProps extends AppProps {
 export default function MyApp(props: MyAppProps) {
   const { Component, emotionCache = clientSideEmotionCache, pageProps } = props;
 
+  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
```
Comment (var-bin) [inline pages/_app.tsx:26]: Let's move this logic + `React.useMemo` into a separate file under a specific directory (e.g. hooks). Last but not least, please, use restructuring:
`import React from 'react';` -> `import { useMemo } from 'react';`

File: pages/_document.tsx [hunk 5]
```diff
@@ -21,8 +25,14 @@ interface MyDocumentProps extends DocumentProps {
 }
 
 export default function MyDocument({ emotionStyleTags }: MyDocumentProps) {
+  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
```
Comment (var-bin) [inline pages/_document.tsx:28]: After implementing my previous comment, we will fix duplication of logic

**Review by var-bin — CHANGES_REQUESTED** [review]

File: darkTheme.ts (hunk 2 above)
Comment (InnaLazarenko) [reply darkTheme.ts:2]: '+

File: lightTheme.ts (hunk 3 above)
Comment (InnaLazarenko) [reply lightTheme.ts:9]: '+

File: .github/workflows/nextjs.yml
Comment (InnaLazarenko) [reply .github/workflows/nextjs.yml:1]: '+

File: pages/_app.tsx (hunk 4 above)
Comment (InnaLazarenko) [reply pages/_app.tsx:26]: '+

File: pages/_document.tsx (hunk 5 above)
Comment (InnaLazarenko) [reply pages/_document.tsx:28]: '+

File: constants/inter-latin.ts (hunk 1 above)
Comment (InnaLazarenko) [reply constants/inter-latin.ts:4]: there is some problem, when I add weight for this font, and it's not required. I left without weight

**Review by var-bin — APPROVED** [review]

**Pull request merged**